lms/djangoapps/verify_student/models.py

```diff
@@ -299,7 +299,7 @@ def user_status(cls, user, window=None):
                     return ('none', error_msg)
 
             if attempt.created_at < cls._earliest_allowed_date():
-                return ('expired', error_msg)
+                return ('expired', _('Your photo verification has expired.'))
```

Comment:
This is the smallest of nitpicks, but we've started trying to use double quotes for strings that are displayed as copy to the user. At least within the team, we seem to be establishing a convention of using single quotes for strings internal to the system.